community/main.py

```diff
@@ -42,268 +42,120 @@ def __init__(self, options):
 		self.COLORED_LIGHTING = options.COLORED_LIGHTING
 		self.ANTIALIASING = options.ANTIALIASING
 
+class IntroScreen(Scene):
+	def __init__(self, window):
+		super(IntroScreen, self).__init__(window, texture=None, color='#FFFFFF')
 
-class Window(pyglet.window.Window):
-	def __init__(self, **args):
-		super().__init__(**args)
-
-		# Options
-		self.options = InternalConfig(options)
-
-		if self.options.INDIRECT_RENDERING and not gl.gl_info.have_version(4, 2):
-			raise RuntimeError("""Indirect Rendering is not supported on your hardware
-			This feature is only supported on OpenGL 4.2+, but your driver doesnt seem to support it, 
-			Please disable "INDIRECT_RENDERING" in options.py""")
-	
-		# F3 Debug Screen
-
-		self.show_f3 = False
-		self.f3 = pyglet.text.Label("", x = 10, y = self.height - 10,
-				font_size = 16,
-				color = (255, 255, 255, 255),
-				width = self.width // 3,
-				multiline = True
-		)
-		self.system_info = f"""Python: {platform.python_implementation()} {platform.python_version()}
-System: {platform.machine()} {platform.system()} {platform.release()} {platform.version()}
-CPU: {platform.processor()}
-Display: {gl.gl_info.get_renderer()} 
-{gl.gl_info.get_version()}"""
-
-		logging.info(f"System Info: {self.system_info}")
-		# create shader
+		self.intro_text = pyglet.text.Label('Probably can\'t show the', font_size=40, font_name=('Verdana', 'Calibri', 'Arial'), x=self.window.width/2, y=self.window.height/2, multiline=False, width=self.window.width, height=self.window.height, color=(255, 165, 0, 255), anchor_x='center')
+		self.has_been_visible_since = time()
 
-		logging.info("Compiling Shaders")
-		if not self.options.COLORED_LIGHTING:
-			self.shader = shader.Shader("shaders/alpha_lighting/vert.glsl", "shaders/alpha_lighting/frag.glsl")
-		else:
-			self.shader = shader.Shader("shaders/colored_lighting/vert.glsl", "shaders/colored_lighting/frag.glsl")
-		self.shader_sampler_location = self.shader.find_uniform(b"u_TextureArraySampler")
-		self.shader.use()
+	def on_draw(self):
+		self.draw()
+		self.intro_text.draw()
+		if time() - 2 > self.has_been_visible_since:
+			self.intro_text.text = 'Mojang Logo due to copyright.'
 
-		# create textures
-		logging.info("Creating Texture Array")
-		self.texture_manager = texture_manager.TextureManager(16, 16, 256)
+class MenuScreen(Scene):
+	def __init__(self, window):
+		image = pyglet.image.load("textures/dirt.png").get_texture()
```

Comment:
Should probably set magnification filtering to `GL_NEAREST` (see [episode 5](https://youtu.be/U9Ldr_PeRA8?t=260)).